src/libraries/pkg/baseline/packageIndex.json

```diff
@@ -4208,10 +4208,10 @@
     },
     "System.Net.Quic": {
       "InboxOn": {
+        "netcoreapp3.0": "4.2.1.0",
         "net5.0": "5.0.0.0"
```

Comment:
I also don't see it on `net5.0`: https://github.com/dotnet/runtime/blob/release/5.0/src/libraries/NetCoreAppLibrary.props